server/repository/migrations/postgres/2022-04-27T13-05_create_document_trigger/up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TYPE changelog_table_name ADD VALUE 'document' AFTER 'clinician_store_join';
+
+CREATE TRIGGER document_trigger
```

Comment:
Will need to create a function for document trigger if you want to use the is_sync_update flag